packages/terra-clinical-onset-picker/src/OnsetPicker.jsx

```diff
@@ -476,17 +489,17 @@ class OnsetPicker extends React.Component {
           </SelectField>
 
           {granularitySelect}
-        </FieldSet>
 
-        {(this.state.precision !== OnsetUtils.PrecisionOptions.UNKNOWN) && (
-          <FieldSet className={cx('fieldset')}>
-            {ageInput}
-            {ageUnitSelect}
-            {monthSelect}
-            {yearSelect}
-            {dateSelect}
-          </FieldSet>
-        )}
+          {(this.state.precision !== OnsetUtils.PrecisionOptions.UNKNOWN) && (
+            <div>
+              {ageInput}
+              {ageUnitSelect}
+              {monthSelect}
+              {yearSelect}
+              {dateSelect}
+            </div>
```

Comment:
Reading the issue description it sounds like there were two possible solutions. Adding a legend or using a div instead. This PR looks like it's doing both. Are both needed?

Reply:
yes we need both legend and div for onset picker. Since there were to FieldSet inside onset picker we replaced second FieldSet with div so that onset picker will be having single FieldSet within it and legend needs to be added for FieldSet since it was missing on onset-picker.